prow/plugins.yaml

```diff
@@ -120,11 +120,17 @@ slack:
     - k8s-ci-robot # future home of tide
     - k8s-merge-robot # submit queue
     - k8s-release-robot # anago
-    - wojtek-t # 1.7 patch release manager
-    - jpbetz # 1.8 patch release manager
-    - mbohlool # 1.9 patch release manager
-    - calebamiles # 1.10 branch manager
-    - lavalamp # feature-serverside-apply "branch manager"
+    branch_whitelist:
+        release-1.7: 
+            - wojtek-t
```

Comment:
we should preserve the comments as to why these users are whitelisted

Reply:
Agreed, comments preserved in 916ec03a0cab8733cf694dc05ae114811345d249